scripts/scil_recognize_multi_bundles.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+    Compute  RecobundlesX (multi-atlas & multi-parameters).
+    The model need to be cleaned and lightweight.
+    Transform should come from ANTs: (using the --inverse flag)
+    AntsRegistration -m MODEL_REF -f SUBJ_REF
+    ConvertTransformFile 3 0GenericAffine.mat 0GenericAffine.npy --ras --hm
+"""
+
+import argparse
+import logging
+import json
+import os
+import random
+import shutil
+
+import coloredlogs
+import numpy as np
+
+from scilpy.io.utils import add_overwrite_arg, assert_inputs_exist
+from scilpy.segment.voting_scheme import VotingScheme
+
+
+def _buildArgsParser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
```

Comment:
Can you add the recobundle reference in the parser ? e.g. `epilog="References: [1] Girard, G., Whittingstall K., Deriche, R., "
               "and Descoteaux, M. (2014). Towards quantitative connectivity "
               "analysis: reducing tractography biases. Neuroimage, 98, "
               "266-278."`